learning-development/r.qmd

```diff
@@ -554,3 +554,10 @@ Then restart R-Studio and downloads with renv should now succeed.
 
 ---
 
+## Using R with ADA/Databricks
```

Comment:
Sorry, missed this before - could we add spaces around the '/' to help with accessibility and text wrapping?